.gitattributes

```diff
@@ -0,0 +1,8 @@
+/driver-testsuite export-ignore
```

Comment:
Well, the builds mostly aren't cached anyways because of API limits (they're cloning as fallback): https://travis-ci.org/minkphp/MinkGoutteDriver/jobs/98220899

But agreed, it would make it harder for them. But it seems a bit odd to include a development dependancy for everyone. As a seperate repository, it wouldn't have to be included for the rest of the users (the 99%)